src/jit/importer.cpp

```diff
@@ -6792,6 +6813,10 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
                 call->gtCall.gtCallObjp = thisPtrCopy;
                 call->gtFlags |= GTF_EXCEPT | (fptr->gtFlags & GTF_GLOB_EFFECT);
 
+                if (((sig->callConv & CORINFO_CALLCONV_GENERIC) != 0) && IsTargetAbi(CORINFO_CORERT_ABI))
```

Comment:
This is the exact same `if` check as above - would it make sense to set a bool flag in the first if check and key off it here? Duplicated if checks are the exact thing that people will forget to update if they ever tweak the first check in the future.